packages/ignite-basic-generators/templates/listview-grid-style.ejs

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import { StyleSheet } from 'react-native'
-import { ApplicationStyles, Metrics, Colors } from '../../Themes/'
+import { ApplicationStyles, Metrics, Colors } from '../../../../../DevScreens/DevTheme/'
```

Comment:
OK.  I'm onboard for that babel root path plugin btw.  Every time I see this now, I cringe.